features/step_definitions/generic_steps.rb

```diff
@@ -238,6 +238,11 @@
   page.body.should =~ /#{Regexp.escape(text)}/m
 end
 
+Then "I should see a link titled {string} to the URL {string}" do |title, path|
```

Comment:
Instead of adding this step, you can test if the link works by following it:
```
When I follow "Evelyn \"Evie\" Carnahan"
Then I should be on the "Evelyn \"Evie\" Carnahan" tag page
```
The quotes may or may not work out though, I'm not sure. But the step definitions are here:
https://github.com/otwcode/otwarchive/blob/8d1456a2b876e5d743c6e71748f8b0cb87f6eb1d/features/step_definitions/web_steps.rb#L46-L50
https://github.com/otwcode/otwarchive/blob/8d1456a2b876e5d743c6e71748f8b0cb87f6eb1d/features/step_definitions/web_steps.rb#L268-L275
https://github.com/otwcode/otwarchive/blob/8d1456a2b876e5d743c6e71748f8b0cb87f6eb1d/features/support/paths.rb#L267-L268

Reply:
Thanks for the input! That'll probably make it easier.

I had to port the fix that allowed double quotes within singly-quoted literals in two places, but that shouldn't be a big problem.